[
  {
    "DefinitionID": "oval:com.ubuntu.focal:def:2017156020000000",
    "Title": "CVE-2017-15602 on Ubuntu 20.04 (focal) - low.",
    "Description": "In GNU Libextractor 1.4, there is an integer signedness error for the chunk size in the EXTRACTOR_nsfe_extract_method function in plugins/nsfe_extractor.c, leading to an infinite loop for a crafted size. It was discovered that Libextractor incorrectly handled integers. An attacker could possibly use this issue to cause a denial of service.",
    "Advisory": {
      "Severity": "Low",
      "Cves": [
        {
          "CveID": "CVE-2017-15602",
          "Cvss2": "",
          "Cvss3": "",
          "Cwe": "",
          "Impact": "",
          "Href": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-15602",
          "Public": ""
        }
      ],
      "Bugzillas": [],
      "AffectedCPEList": [],
      "Issued": "1000-01-01T00:00:00Z",
      "Updated": "1000-01-01T00:00:00Z"
    },
    "Debian": null,
    "AffectedPacks": [
      {
        "Name": "libextractor",
        "Version": "1:1.6-2",
        "Arch": "",
        "NotFixedYet": false,
        "ModularityLabel": ""
      }
    ],
    "References": [
      {
        "Source": "CVE",
        "RefID": "CVE-2017-15602",
        "RefURL": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-15602"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "http://people.canonical.com/~ubuntu-security/cve/2017/CVE-2017-15602.html"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "http://lists.gnu.org/archive/html/bug-libextractor/2017-10/msg00005.html"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://ftp.gnu.org/gnu/libextractor/libextractor-1.6.tar.gz"
      },
      {
        "Source": "Ref",
        "RefID": "",
        "RefURL": "https://ubuntu.com/security/notices/USN-4641-1"
      }
    ]
  }
]
